utils/strings/formatters.ts

```diff
@@ -0,0 +1,29 @@
+export function removeProtocolAndWwwIfPresent(url: string) {
+  const httpProtocolPrefix = 'http://';
+  const httpsProtocolPrefix = 'https://';
+  const wwwPrefix = 'www.';
+
+  let trimmedUrl = url;
+
+  if (trimmedUrl.startsWith(httpProtocolPrefix)) {
+    trimmedUrl = trimmedUrl.slice(httpProtocolPrefix.length);
+  }
+
+  if (trimmedUrl.startsWith(httpsProtocolPrefix)) {
+    trimmedUrl = trimmedUrl.slice(httpsProtocolPrefix.length);
+  }
+
+  if (trimmedUrl.startsWith(wwwPrefix)) {
+    trimmedUrl = trimmedUrl.slice(wwwPrefix.length);
+  }
+
+  return trimmedUrl;
```

Comment:
Nit: I think its simple enough for regex but if you wanna avoid it that's fine too.

```suggestion
  const pattern = /(http[s]?:\/\/)?(www.)?(.+)/;
  const match = url.match(pattern);
  return match.length === 4 ? match[3] : url;
```

Reply:
I think I'm not adopting this in order to avoid regex as much as possible. Regex is less understandable than logic.